src/status_im/utils/handlers.cljs

```diff
@@ -100,7 +100,7 @@
          (let [event    (get-coeffect context :event)
                offline? (or (= :offline (:network-status new-db))
                             (= :offline (:sync-state new-db)))
-               anon-id  (ethereum/sha3 current-account-id)]
```

Comment:
`ethereum` ns declaration can be removed. Same for `current-account-id` definition.

Reply:
done